website/docs/r/nsxt_edgegateway_rate_limiting.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_rate_limiting"
+sidebar_current: "docs-vcd-resource-nsxt-edge-rate-limiting"
+description: |-
+  Provides a resource to manage NSX-T Edge Gateway Rate Limiting (QoS) configuration.
+---
+
+# vcd\_nsxt\_edgegateway\_rate\_limiting
+
+Supported in provider *v3.9+* and VCD 10.3.2+ with NSX-T.
+
+Provides a resource to manage NSX-T Edge Gateway Rate Limiting (QoS) configuration.
```

Comment:
Very remotely related to this PR, but it made me look at the "main" doc page we have for an edge gateway:
https://registry.terraform.io/providers/vmware/vcd/latest/docs/resources/edgegateway

What if we add "NSX-V" in the first sentence of that page to improve users experience? Right now, if someone new is navigating the docs, it's quite easy to incorrectly assume that that page is for NSX-T (given how popular it became).

> Provides a VMware Cloud Director **_NSX-V_** edge gateway directly connected to one or more external networks. This can be used to create and delete edge gateways for Org VDC networks to connect.

Reply:
This doesn't cost a thing - added.

Reply:
Thank you!